docs/index.md

```diff
@@ -779,73 +779,77 @@ Mocha supports the `err.expected` and `err.actual` properties of any thrown `Ass
+--harmony<_classes,_generators,...>     all node --harmony* flags are available
+--preserve-symlinks                     Instructs the module loader to preserve symbolic links when resolving and caching modules
+--icu-data-dir                          include ICU data
+--inline-diffs                          display actual/expected differences inline within each string
+--no-diff                               do not show a diff on failure
+--inspect                               activate devtools in chrome
+--inspect-brk                           activate devtools in chrome and break on the first line
+--interfaces                            output provided interfaces and exit
+--no-deprecation                        silence deprecation warnings
+--exit                                  force shutdown of the event loop after test run: mocha will call process.exit
+--no-timeouts                           disables timeouts, given implicitly with --debug/--inspect
+--no-warnings                           silence all node process warnings
+--opts <path>                           specify opts path (default: "test/mocha.opts")
+--perf-basic-prof                       enable perf linux profiler (basic support)
+--napi-modules                          enable experimental NAPI modules
+--prof                                  log statistical profiling information
+--log-timer-events                      Time events including external callbacks
+--recursive                             include sub directories
+--reporters                             output provided reporters and exit
+--retries <times>                       specify number of times to retry a failed test case (default: 0)
+--throw-deprecation                     throw an exception anytime a deprecated function is used
+--trace                                 trace function calls
+--trace-deprecation                     show stack traces on deprecations
+--trace-warnings                        show stack traces on node process warnings
+--use_strict                            enforce strict mode
+--watch-extensions <ext>,...            specify extensions to monitor with --watch (default: ["js"])
+--delay                                 wait for async suite definition
+--allow-uncaught                        enable uncaught errors to propagate
+--forbid-only                           causes test marked with only to fail the suite
+--forbid-pending                        causes pending tests and test marked with skip to fail the suite
+--file <file>                           adds file be loaded prior to suite execution (default: [])
+--exclude <file>                        adds file or glob pattern to ignore (default: [])
+-h, --help                              output usage information
 
 Commands:
-  init <path>                             initialize a client-side mocha setup at <path>
+init <path>                             initialize a client-side mocha setup at <path>
 ```
```

Comment:
Why was indentation removed? Running `--help` on most programs displays cmdline options with indentation.

Reply:
probably has to do with the lack of a tty somewhere.  

the actual `bin/mocha --help` output has indents.

Reply:
That's what I thought this was supposed to do -- run `mocha --help` and paste its _exact_ content.